docs/support.md

```diff
@@ -3,17 +3,66 @@ hide:
+      our On-Call Support Engineer will respond within 30 minutes. Example: Cirrus CI showing 502 errors for all users.
+
+!!! info "How to submit a priority issue"
+    Once your organization [signs the Priority Support Subscription contract](#how-to-purchase-priority-support-subscription)
+    members of your organization will get access to a separate support email specified in your subscription contract.
+
+## Priority Support Pricing
+
+As a company grows, engineering team tend to accumulate knowledge operating and working with Cirrus CI,
+therefore there is less need of "support effort per seat" from our side. In addition, Cirrus CI allows to [bring your own infrastructure](guide/supported-computing-services.md)
+which increases complexity of the support. As a result we reflected the above challenges in a progressive
+pricing model based on a seat amount and a type of infrastructure used:
+
+| Seat Amount | Only [managed by us instance types](guide/writing-tasks.md#execution-environment) | [Bring your own infrastructure](guide/supported-computing-services.md) |
+|-------------|-----------------------------------------------------------------------------------|------------------------------------------------------------------------|
+| 20-100      | $60/seat/month                                                                    | $100/seat/month                                                        |
+| 101-300     | $45/seat/month                                                                    | $75/seat/month                                                         |
+| 301-500     | $30/seat/month                                                                    | $50/seat/month                                                         |
+| 500+        | $15/seat/month                                                                    | $25/seat/month                                                         |
+
+??? info "What is a seat?"
+    A seat is a user that initiates CI builds by pushing commits and/or creating pull requests in a **private** repository.
+    It can be a real person or a bot. If you are using [Cron Builds](guide/writing-tasks.md#cron-builds) or creating builds through [Cirrus's API](api.md)
+    it will be counted as an additional seat (like a bot).
+
+    If you'd like to get a priority support for your public repositories then the amount of seats will be equal to the amount of members in your organization.
+
+## How to purchase Priority Support Subscription
+
+Please email [sales@cirruslabs.org](mailto:sales@cirruslabs.org), so we can get a support contract in addition to [TOC](legal/terms.md).
+The contract will contain a special priority email address for your organization and other helpful information. Sales team will
+also schedule a checkin meeting to make sure your engineering team is set for success and Cirrus CI roadmap aligns with your needs.
```

Comment:
Not sure about this one, but if this is correct, it's probably a good idea to check other places with the `checkin` usage in the document too.

```suggestion
also schedule a check-in meeting to make sure your engineering team is set for success and Cirrus CI roadmap aligns with your needs.
```